tools/tool-env/isopackets.js

```diff
@@ -229,8 +229,6 @@ var newIsopacketBuildingCatalog = function () {
       isopacketCatalog.initialize({
         localPackageSearchDirs: [
           files.pathJoin(files.getCurrentToolsDir(), 'packages'),
-          files.pathJoin(files.getCurrentToolsDir(),
-            'packages', 'non-core', 'blaze', 'packages')
```

Comment:
@tmeasday If this works, it means we don't have to hard-code `blaze` in tools code, which is great news!